include/config/item.h

```diff
@@ -18,7 +18,7 @@
 #define I_REUSABLE_TMS          FALSE       // In Gen5-8, TMs are reusable. Setting this to TRUE will make all vanilla TMs reusable, though they can also be cherry-picked by setting their importance to 1.
 
 // Exp. Share config
-#define I_EXP_SHARE_FLAG        0           // If this flag is set, every Pokémon in the party will gain experience, regardless if they participated in the battle or not.
+#define I_EXP_SHARE_FLAG        0 // If this flag is set, every Pokémon in the party will gain experience, regardless if they participated in the battle or not.
```

Comment:
Was there a reason to misalign the comment like this? :eyes:

Reply:
Nope, it's from me testing it. I can change it back